proto/pulumi/analyzer.proto

```diff
@@ -20,7 +20,7 @@ import "google/protobuf/struct.proto";
 
 package pulumirpc;
 
-option go_package = "github.com/pulumi/pulumi/v3/proto/go/pulumirpc";
+option go_package = "github.com/pulumi/pulumi/sdk/v3/proto/go;pulumirpc";
```

Comment:
I'm assuming that the `;` means import as.

Reply:
The compiler uses it to set the package option at the top of the generated files. If your missing it you get "package _go".